Doc/howto/a-conceptual-overview-of-asyncio.rst

```diff
@@ -540,8 +539,8 @@ will monitor how much time has elapsed and, accordingly, call
        # Block until the future is marked as done.
        await future
 
-Below, we'll use a rather bare object, ``YieldToEventLoop()``, to ``yield``
-from ``__await__`` in order to cede control to the event loop.
+Below, we use a simple ``YieldToEventLoop()`` object to ``yield``
```

Comment:
I like this rearrangement. But I prefer "bare" to "simple". I think "bare" more accurately describes the object. I also generally try to avoid "simple" in docs as it often doesn't communicate much besides the writers perspective and may make the user feel bad if they don't get it. And arguably this is not a simple object, just one with few lines of code.